app/graphql/types/candidate_type.rb

```diff
@@ -0,0 +1,25 @@
+module Types
+  class CandidateType < Types::BaseObject
+    field :id, ID, null: false do
```

Comment:
For each argument for a CandidateType I added a description so I would be able to see/remind myself what each argument was for when I saw them within the grapiql interface.